docs/host-powershell/README.md

```diff
@@ -2,155 +2,23 @@
-Once the execution starts this way, all further assembly loading requests will be handled by the PowerShell load context.
-
-So the above example needs to be altered as follows in a .NET Core application:
-
-```CSharp
-// MyApp.exe
-using System.Management.Automation;
-using System.Reflection;
-
-namespace Application.Test
-{
-    public class Program
-    {
-        /// <summary>
-        /// Managed entry point shim, which starts the actual program
-        /// </summary>
-        public static int Main(string[] args)
-        {
-            // Application needs to use PowerShell AssemblyLoadContext if it needs to create PowerShell runspace
-            // PowerShell engine depends on PS ALC to provide the necessary assembly loading/searching support that is missing from .NET Core
-            string appBase = System.IO.Path.GetDirectoryName(typeof(Program).GetTypeInfo().Assembly.Location);
-            System.Console.WriteLine("\nappBase: {0}", appBase);
-
-            // Initialize the PS ALC and let it load 'Logic.dll' and start the execution
-            return (int)PowerShellAssemblyLoadContextInitializer.
-                           InitializeAndCallEntryMethod(
-                               appBase,
-                               new AssemblyName("Logic, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"),
-                               "Application.Test.Logic",
-                               "Start",
-                               new object[] { args });
-        }
-    }
-}
-
-// Logic.dll
-using System;
-using System.Management.Automation;
-using System.Management.Automation.Runspaces;
-
-namespace Application.Test
-{
-    public sealed class Logic
-    {
-        /// <summary>
-        /// Start the actual logic
-        /// </summary>
-        public static int Start(string[] args)
-        {
-            // My business logic code
-            using (PowerShell ps = PowerShell.Create())
-            {
-                var results = ps.AddScript("Get-Command Write-Output").Invoke();
-                Console.WriteLine(results[0].ToString());
-            }
-            return 0;
-        }
-    }
-}
-```
-
-[CorePsAssemblyLoadContext.cs]: https://github.com/PowerShell/PowerShell/blob/v6.0.0-beta.2/src/Microsoft.PowerShell.CoreCLR.AssemblyLoadContext/CoreCLR/CorePsAssemblyLoadContext.cs
-[Resolving]: https://github.com/dotnet/corefx/blob/ec2a6190efa743ab600317f44d757433e44e859b/src/System.Runtime.Loader/ref/System.Runtime.Loader.cs#L35
-[bootstrapping]: https://github.com/PowerShell/PowerShell/blob/v6.0.0-beta.2/src/powershell/Program.cs#L27
+Please see the [.NET Core Sample Application](#net-core-sample-application) section for an example that uses PowerShell Core NuGet packages.
 
+[CorePsAssemblyLoadContext.cs]: https://docs.microsoft.com/dotnet/api/system.management.automation.powershellassemblyloadcontextinitializer.setpowershellassemblyloadcontext
```

Comment:
Is it still used?

Reply:
Yes, it's still used by the DSC native host that is used in Azure. The DSC host doesn't have the powershell assemblies in its TPA list, and thus is loading powershell in a different way.

Reply:
Well, I think I misunderstood your comment :) No, I don't find this link used anywhere in this doc after the change.

Reply:
But we need to keep the introduction about `SetPowerShellAssemblyLoadContext` as it's needed for a special hosting scenario.

Reply:
Ok, I brought those docs back in [this](https://github.com/PowerShell/PowerShell/pull/8472/commits/98551c83be5cdf737dca4ac72b494ac9142b4c44) self-contained commit